src/components/environment-banner/environment-banner.css

```diff
@@ -0,0 +1,17 @@
+.environment-banner-section {
+    color: #333;
+}
+
+.environment-banner-title {
```

Comment:
After discussing with @Mr0grog I think it is ok to have the class names be shorter since they are suffixed with a hash anyway and don't need the name scoping. So `.title` would be preferable. Sorry to have you to redo this, but it should be a quick fix.

Reply:
for sure @SYU15 , I can make these changes. I will let you know when I've made the commit

Reply:
Okay, just made the commit with the changes.